src/Tronkell/Server/Server.hs

```diff
@@ -38,12 +39,14 @@ startServer gConfig = do
     moveAllPlayersToWaiting (serverUsers server)
     gameEngineThread server Nothing
 
-  -- start websockets
-  wsThread <- forkIO $ W.start firstUId networkChans clientsChan
+  -- start network
+  wsThread  <- forkIO $ W.start firstUId networkChans clientsChan
+  tcpThread <- forkIO $ Tcp.start firstUId networkChans clientsChan
```

Comment:
I guess this is the thing that can become our "interface" for transports at some point.